app/os_windows.go

```diff
@@ -662,9 +691,9 @@ func (w *window) Configure(options []Option) {
 	swpStyle := uintptr(windows.SWP_NOZORDER | windows.SWP_FRAMECHANGED)
 	winStyle := uintptr(windows.WS_OVERLAPPEDWINDOW)
 	style &^= winStyle
-	if !w.config.Decorated {
-		winStyle = 0
-	}
+	//if !w.config.Decorated {
+	//	winStyle = 0
+	//}
```

Comment:
Delete. Judging from your video, I assume switching between undecorated/decorated still works?

Reply:
Yes, it still works. Now the WM_NCCALCSIZE message controls how to show or hide the title bar.

Reply:
See: https://learn.microsoft.com/en-us/windows/win32/dwm/customframe

Reply:
Thanks. Please delete the disabled lines completely.